packages/side-nav/src/AccessibleGlyph/AccessibleGlyph.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { CollapsedSideNavItem } from '../CollapsedSideNavItem/CollapsedSideNavItem';
+
+import { AccessibleGlyphProps } from './types';
+
+/**
+ * @internal
+ */
+export function AccessibleGlyph({
+  isActiveGroup,
+  className,
+  accessibleGlyph,
+}: AccessibleGlyphProps) {
+  return (
+    <>
+      {accessibleGlyph && (
```

Comment:
I'm curious why this check happens here. I feel like this should be checked in whatever parent component is called this component, and the `accessibleGlyph` prop should just be required

Reply:
That makes sense. I'll update!